content/ko/docs/concepts/cluster-administration/networking.md

```diff
@@ -30,46 +30,7 @@ weight: 50
 찾는 방법 등을 알아야 한다. 쿠버네티스는 이런 것들을 다루는 대신
 다른 접근법을 취한다.
 
-## 쿠버네티스 네트워크 모델
```

Comment:
Note: L33-72 was migrated to another document - will address via another issue/PR